frontend/components/BadgeFilter.vue

```diff
@@ -163,14 +158,20 @@ export default {
   width: fit-content;
   border-radius: 1em;
   background-color: #ededed;
-  margin: 0.25em 0.125em;
+  margin: 0.25em 0.125em !important;
```

Comment:
We avoid using `!important` as written in the ops manual, but I'm pretty sure you have a good reason to do this seeing that you replaced the old one without `!important`.

Just FYI, here's the ops that mentions that: https://github.com/countable-web/ops-manual/blob/master/developers/HTML_CSS.rst#css-coding-standards